saleor/graphql/core/types/shop.py

```diff
@@ -0,0 +1,119 @@
+        required=True)
+    currencies = graphene.List(
+        graphene.String, description='List of available currencies.',
+        required=True)
+    default_currency = graphene.String(
+        description='Default shop\'s currency.', required=True)
+    description = graphene.String(description='Shop\'s description.')
+    domain = graphene.Field(
+        Domain, required=True, description='Shop\'s domain data.')
+    languages = graphene.List(
+        LanguageDisplay,
+        description='List of the shops\'s supported languages.', required=True)
+    name = graphene.String(description='Shop\'s name.', required=True)
+    permissions = graphene.List(
+        PermissionDisplay, description='List of available permissions.',
+        required=True)
+    phone_prefixes = graphene.List(
+        graphene.String, description='List of possible phone prefixes.',
+        required=True)
+    tax_rates = graphene.List(
+        VAT, description='List of VAT tax rates configured in the shop.',
+        required=True)
+    tax_rate = graphene.Field(
+        VAT, description='VAT tax rates for a specific country.',
+        required=True, country_code=graphene.Argument(graphene.String))
+
+    class Meta:
+        description = '''
+        Represents a shop resource containing general shop\'s data
+        and configuration.'''
+
+    @permission_required('site.manage_settings')
+    def resolve_authorization_keys(self, info):
+        return site_models.AuthorizationKey.objects.all()
+
+    def resolve_countries(self, info):
+        return [
+            CountryDisplay(code=country[0], country=country[1])
+            for country in countries]
+
+    def resolve_currencies(self, info):
+        return settings.AVAILABLE_CURRENCIES
+
+    def resolve_domain(self, info):
+        site = info.context.site
+        return Domain(
+            host=site.domain,
+            ssl_enabled=settings.ENABLE_SSL,
+            url=info.context.build_absolute_uri('/'))
+
+    def resolve_default_currency(self, info):
+        return settings.DEFAULT_CURRENCY
+
+    def resolve_description(self, info):
+        return info.context.site.settings.description
+
+    def resolve_languages(self, info):
+        return [
+            LanguageDisplay(code=language[0], language=language[1])
+            for language in settings.LANGUAGES]
+
+    def resolve_name(self, info):
+        return info.context.site.name
+
+    @permission_required('site.manage_settings')
+    def resolve_permissions(self, info):
+        permissions = get_permissions()
+        return format_permissions_for_display(permissions)
+
+    def resolve_phone_prefixes(self, info):
+        return list(COUNTRY_CODE_TO_REGION_CODE.keys())
+
+    @permission_required('site.manage_settings')
+    def resolve_tax_rates(self, info):
+        return vatlayer_models.VAT.objects.order_by('country_code')
+
+    @permission_required('site.manage_settings')
```

Comment:
I wonder if we need to set up permission check here. How is information about tax rates (I guess they are available on some economic law websites) more restricted to users than (crypto)currencies, served countries or phone prefixes used by shop?

Reply:
We fetch tax rates from Vatlayer API which except for simple basic plan has also paid ones. I think we shouldn't expose this data to the world as someone might use our API to get data the others have to pay for.

Reply:
I second @maarcingebala's sentiment here, while tax rates are public domain information we wouldn't want someone to start using our demo site as a data source for a third party app.